frontend/components/ArticleBody.tsx

```diff
@@ -366,8 +366,13 @@ const ArticleBody: React.SFC<{
                         </li>
                     </ul>
                     <ShareCount config={config} CAPI={CAPI} />
+
                     <div className={ageWarning(pillarColour)}>
-                        <ClockIcon /> This article is over 1 year old.
+                        {CAPI.ageWarning && (
```

Comment:
Does this check belong outside the containing `div`?

Reply:
Yes!